tests/test_force_https.py

```diff
@@ -0,0 +1,32 @@
+from tests.helpers import BaseApplicationTest, Config
+from dmutils import config, force_https
+from flask import Flask
+
+
+class TestForceHTTPSConfig(Config):
+    DM_HTTP_PROTO = "HTTPS"
+
+
+class TestForceHTTPS(BaseApplicationTest):
+    config_object = TestForceHTTPSConfig()
+
+    def setup(self):
+        self.flask = Flask(
+            'test_app'
+        )
+        self.flask.config.from_object(self.config_object)
+        config.init_app(self.flask)
+        force_https.init_app(self.flask)
+        self.app = self.flask.test_client()
+
+        @self.flask.route('/some-page')
+        def some_page():
+            return 'Interesting content'
+
+    def test_http_view(self):
+        res = self.app.get('/some-page')
+        assert res.status_code == 301
+
+    def test_https_view(self):
+        res = self.app.get('/some-page', base_url="https://localhost")
+        assert res.status_code == 200
```

Comment:
Cool, you got a way to test this.  How about adding a check for the HSTS header?  (I wouldn't bother checking the value itself though, just that it got inserted as intended.)